docs/userguide/networking/dockernetworks.md

```diff
@@ -0,0 +1,368 @@
+          inet6 addr: ::1/128 Scope:Host
+          UP LOOPBACK RUNNING  MTU:65536  Metric:1
+          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:0
+          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
+
+/ #
+```
+**Note**: You can detach from the container and leave it running with `CTRL-p CTRL-q`.
+
+The `host` network adds a container on the hosts network stack. You'll find the network configuration inside the container is identical to the host.
+
+With the exception of the the `bridge` network, you really don't need to interact with these default networks. While you can list and inspect these networks, you cannot remove them. They are required by your Docker installation.  
+However, you can add to the list of networks by creating new ones yourself. Before you learn more about creating your own networks, it is worth looking at the `default` network a bit.
+
+
+### The default bridge0 network in detail
+The default `bridge` network is present on all Docker hosts. First, inspect the `bridge` network:
+
+```
+$ sudo docker network inspect bridge
+{
+    "name": "bridge",
+    "id": "7fca4eb8c647e57e9d46c32714271e0c3f8bf8d17d346629e2820547b2d90039",
+    "driver": "bridge",
+    "containers": {}
+}
+```
+
+Start two new containers running `busybox`, `container1` and `container2` respectively.
+
+```
+$ sudo docker run -itd --name=container1 busybox
+f2870c98fd504370fb86e59f32cd0753b1ac9b69b7d80566ffc7192a82b3ed27
+
+$ sudo docker run -itd --name=container2 busybox
+bda12f8922785d1f160be70736f26c1e331ab8aaf8ed8d56728508f2e2fd4727
+```
+
+And then inspect the `bridge` network again. You can recognize both newly launched containers by their ids inside the network.
+
+```
+$ sudo docker network inspect bridge
+{
+    "name": "bridge",
+    "id": "7fca4eb8c647e57e9d46c32714271e0c3f8bf8d17d346629e2820547b2d90039",
+    "driver": "bridge",
+    "containers": {
+        "bda12f8922785d1f160be70736f26c1e331ab8aaf8ed8d56728508f2e2fd4727": {
+            "endpoint": "e0ac95934f803d7e36384a2029b8d1eeb56cb88727aa2e8b7edfeebaa6dfd758",
+            "mac_address": "02:42:ac:11:00:03",
+            "ipv4_address": "172.17.0.3/16",
+            "ipv6_address": ""
+        },
+        "f2870c98fd504370fb86e59f32cd0753b1ac9b69b7d80566ffc7192a82b3ed27": {
+            "endpoint": "31de280881d2a774345bbfb1594159ade4ae4024ebfb1320cb74a30225f6a8ae",
+            "mac_address": "02:42:ac:11:00:02",
+            "ipv4_address": "172.17.0.2/16",
+            "ipv6_address": ""
+        }
+    }
+}
+```
+`docker network inspect` command above shows all the connected containers and its network resources on a given network
+
+Containers in this default network are able to communicate with each other using container names.  First `attach` to your running `container` and investigate its  configuration:
+
+```
+$ sudo docker attach container1
```

Comment:
Wondering if we should use `docker attach` in these examples, because that attaches to the running process; people that started (e.g.) an `nginx` container may try this, and that won't work (because you're attaching to the `nginx` process, not a shell)

Using `docker exec -it container sh` may be a better alternative, as it should always open a shell session (given the container has a shell of course)